ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-clo5-cleanup.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
@imiller0 Not related to this CR, but was wondering if we should update any readme under acmpolicygenerator or in the [argod folders .md file](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd) have a small section explaining how this policy and the file under source-cr named ClusterLogging5Cleanup.yaml should be used, although the comments on the CR explains that